package.json

```diff
@@ -33,17 +33,17 @@
     "package.json": "sort-package-json"
   },
   "dependencies": {
-    "isolmate": "github:defi-wonderland/isolmate#59e1804"
+    "solmate": "github:transmissions11/solmate#a9e3ea2"
   },
   "devDependencies": {
-    "@commitlint/cli": "17.0.3",
-    "@commitlint/config-conventional": "17.0.3",
-    "@defi-wonderland/natspec-smells": "1.0.3",
+    "@commitlint/cli": "19.3.0",
+    "@commitlint/config-conventional": "19.2.2",
+    "@defi-wonderland/natspec-smells": "1.1.1",
     "ds-test": "github:dapphub/ds-test#e282159",
-    "forge-std": "github:foundry-rs/forge-std#4513bc2",
+    "forge-std": "github:foundry-rs/forge-std#bb4ceea",
     "husky": ">=8",
     "lint-staged": ">=10",
-    "solhint": "github:solhint-community/solhint-community#v4.0.0-rc01",
-    "sort-package-json": "1.53.1"
+    "solhint": "github:solhint-community/solhint-community#v4.0.0",
```

Comment:
Now that there is a released version, could you update the source to install it from NPM?
https://www.npmjs.com/package/solhint-community

Reply:
sure!